scss/objects/_wrap.scss

```diff
@@ -6,7 +6,7 @@ $DBwrap-namespace: "o-wrap";
 $DBwrap-width: 990px;
 
 .#{$DBwrap-namespace} {
-  display: block;
-  max-width: $DBwrap-width;
-  margin: 0 auto;
+    display: block;
+    max-width: $DBwrap-width;
```

Comment:
Name of variable `DBwrap-width` should be written in all lowercase letters with hyphens instead of underscores